### ISSUE TITLE
Overflow issue with the Stripe SignInView component.

### ISSUE THREAD

**Issue by Matthieu-OD** [stripe/stripe-apps#779]
**Describe the bug**
When using the SignInView component on my app, I sometimes have scaling issues. Not on the first rendering but later on. I tried to add Box components to fix the width but it's not working.

<img width="330" alt="image (1)" src="https://user-images.githubusercontent.com/59685483/208411518-4a1e5979-cf68-45f8-8804-b9b1f722ac53.png">

For now, I didn't this it on other views. A way to fix this is to open the app in a new tab and it will scale properly.

**Expected behavior**
I would like to have my app scaling correctly to the drawer when using the SignInView component.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Brave
 - Version 
<img width="425" alt="image" src="https://user-images.githubusercontent.com/59685483/208411251-4e81f438-fd23-4f82-947d-36df131fd546.png">

**Comment by Matthieu-OD** [comment]
Happy years, any news about this one?

**Comment by Matthieu-OD** [comment]
I'm only having this with the SignInView component. When I'm using the ContextView, everything is going fine.

In my code, I'm just returning the SignInView component. I also tried to add a box element to limit the component size, but it didn't work.

**Comment by Matthieu-OD** [comment]
From what I've tried, it's related to the width of the `SignInView` component. It's not something I can control.

First, I changed my code to the one provided in the stripe documentation. I still had the same result.

<img width="1242" alt="image" src="https://user-images.githubusercontent.com/59685483/210323389-612e22e0-8c3b-479d-902d-de99a6c487e3.png">

I also tried to add a `Box` component to limit the width of the `SignInView` component to the width of the drawer. Again, it didn't work.


I inspected the drawer to ensure it comes from the`SignInView` component (see gif).

![signin_component_overflowing](https://user-images.githubusercontent.com/59685483/210327967-f1024fde-e18d-48af-9bcd-91b1626eeeeb.gif)

I also saw that the overflow issue happens when we are on certain pages. For example, we can see it on the connect tab but not on the home tab (see gif).

![overflow_on_connect_page](https://user-images.githubusercontent.com/59685483/210328086-36dd55a6-8599-493d-a535-dff76ccde29b.gif)
